common/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/common/model/SyncIndex.kt

```diff
@@ -5,11 +5,28 @@ enum class SyncIndex(val indexName: String) {
   GREEN("prisoner-search-green"),
   BLUE("prisoner-search-blue"),
   NONE("new-indexes"),
+
+  GREEN_I("incentive-search-green"),
+  BLUE_I("incentive-search-blue"),
+  NONE_I("new-incentive-indexes"),
```

Comment:
3 new values to reflect incentives state. Could improve names later.
I tried a second SyncIndex class with inheritance but hit a problem deserialising the status index's currentIndex value from a string to the appropriate class.

Reply:
If all indexes were tied together instead of managed independently then would `val indexName: String` become `val indexNames: List<String>`?

Reply:
Yep, or I could add a 'incentiveIndexName' etc so the same 3 enums could be used for all indices and they could remain independent